Services/Notifications/classes/class.ilNotificationGUI.php

```diff
@@ -107,18 +107,7 @@ private function getAvailableChannels(array $types = []): array
 
     public function getOSDNotificationsObject(): void
     {
-        $settings = new ilSetting('notifications');
-        ilSession::enableWebAccessWithoutSession(true);
```

Comment:
Please don't remove `ilSession::enableWebAccessWithoutSession(true);` here and elsewhere.

I know this is ugly but the only way to prevent the user session rewritten to database and extended in async. calls.